.travis.yml

```diff
@@ -31,5 +33,8 @@ script:
   - cd lighthouse-extension
   - gulp build -- $__node_harmony
   - cd ..
+  - cd lighthouse-viewer
+  - gulp
```

Comment:
for future reference, you might need to add `-- $__node_harmony` (like the extension does) if you end up using fancier features in your build process